src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -370,19 +369,6 @@ pub struct Args {
     persist_isolated_runtime_threads: Option<isize>,
 
     // === Adapter options. ===
-    /// The PostgreSQL URL for the adapter stash.
-    #[clap(
```

Comment:
We should make sure we remove these CLI args from the cloud side too, not totally sure what happens if we provide unrecognized arguments but I think it'll cause an error

Reply:
Good point, I have https://github.com/MaterializeInc/cloud/pull/8888 which removes the upgrade checker CLI args, but I forgot about this one. I'll extent the PR to include this.